src/utils.tsx

```diff
@@ -0,0 +1,6 @@
+import { chakra } from "@chakra-ui/react";
+import NextImage from "next/image";
+
+export const Image = chakra(NextImage, {
+  shouldForwardProp: prop => ['width', 'height', 'src', 'alt'].includes(prop),
+});
```

Comment:
Does it make sense to place this in the components folder?

Reply:
Maybe, yeah. Give me a few!